src/app/collective-rewards/rewards/backers/Rewards.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from 'react'
+import {
+  BackerAllTimeShare,
+  BackerClaimableRewards,
+  useClaimBackerRewards,
+} from '@/app/collective-rewards/rewards/backers'
+import {
+  RewardDetails,
+  useClaimStateReporting,
+  BackerRewardsCard,
+  BackerRewardsContextProvider,
+} from '@/app/collective-rewards/rewards'
+import { CycleContextProvider } from '@/app/collective-rewards/metrics'
+import { PricesContextProvider } from '@/shared/context/PricesContext'
+import { Popover } from '@/components/Popover'
+import { Button } from '@/components/Button'
+import { Paragraph } from '@/components/Typography'
+
+type RewardsProps = RewardDetails
+
+export const Rewards: FC<RewardsProps> = ({ builder, gauges, tokens }) => {
+  const { claimRewards, ...claimTx } = useClaimBackerRewards(gauges)
+
+  useClaimStateReporting({ ...claimTx })
+
+  return (
+    <div className="grid grid-cols-4 gap-[16px]">
```

Comment:
NIT: I'd use flex as specified in the design, but I haven't seen the visual result yet.